earth_enterprise/src/installer/install_fusion.sh

```diff
@@ -822,10 +822,10 @@ install_or_upgrade_asset_root()
         if [ $IS_SLAVE == false ]; then
             # TODO: Verify this logic -- this is what is defined in the installer documentation, but need confirmation
             if [ $NEW_GEGROUP == true ] || [ $NEW_GEFUSIONUSER == true ]; then
-                NOCHOWN=""
+                DOCHOWN="--chown"
```

Comment:
Should we add similar checks in install_server.sh for consistency sake?

Reply:
I hadn't made any logic changes to the regular installers, but it would be nice to have consistency between the RPM and non-RPM installers.

Since we decided to remove the privilege changes for the publish root from the RPM installer, I'll remove the `--chown` here and update message, and also remove the `chown -R` calls in `earth_enterprise/src/installer/install_server.sh`

That way all of the installers should be consistent about not changing permissions and slowing down installation.